blackbelt/handle_github.py

```diff
@@ -160,24 +160,20 @@ def merge(pr_url):
     """
 
     pr_info = get_pr_info(pr_url)
-
-    pr_api_url = "https://api.github.com/repos/%(owner)s/%(name)s/pulls/%(number)s" % pr_info
-
-    headers = {
-        'Authorization': "token %s" % config['github']['access_token'],
-        'User-Agent': UA_STRING
-    }
-
-    r = requests.get(pr_api_url, headers=headers)
-
-    if (r.status_code != 200):
-        raise ValueError("Cannot retrieve PR info with status code %s: %s" % (r.status_code, r))
-
-    pr = r.json()
-
+    pr = get_json_response("https://api.github.com/repos/%(owner)s/%(name)s/pulls/%(number)s" % pr_info)
```

Comment:
@abtris this way we make sure all required checks are passing (no conflicts with base, different CIs, security checks, etc.)